### DISCUSSION TITLE
Is it possible to have an uneven number of workspaces in each column of the grid?

### DISCUSSION THREAD

**Discussion by joelostblom** [mzur/gnome-shell-wsmatrix#261] [Q&A] [answered]
I often find myself lost between different projects being spread out across workspaces. I would like to arrange my workspaces to have one column of virtual desktops per project, and that each column allows me to add new desktops on top and bottom. Is this possible with this extension?

An example layout could look like this:

```
Empty    Empty           Empty
Work     HobbyProject    Personal
Work     HobbyProject    Personal
Work     Empty           Personal
Work                     Empty
Empty
```

**Comment by mzur** [comment, marked as answer]
This is not possible with this extension, sorry. Have a look at [PaperWM](https://github.com/paperwm/PaperWM). There, the Workspaces expand horizontally with added windows and you have vertical workspaces.

**Reply by joelostblom** [reply]
Thanks for the tip!
